src/Nethermind/Baseline.sln

```diff
@@ -85,7 +85,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Nethermind.Network.Stats",
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Nethermind.PubSub", "Nethermind.PubSub\Nethermind.PubSub.csproj", "{A913DB39-865B-4162-B279-16654B20D949}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Nethermind.WebSockets", "Nethermind.WebSockets\Nethermind.WebSockets.csproj", "{3C731BA7-F932-4771-A890-7FB4AB10F80F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Nethermind.DataMarketplace.WebSockets", "Nethermind.DataMarketplace.WebSockets\Nethermind.DataMarketplace.WebSockets.csproj", "{3C731BA7-F932-4771-A890-7FB4AB10F80F}"
```

Comment:
This looks wrong Baseline has nothing to do with DataMarketplace

Reply:
While testing it out, there was no `Nethermind.WebSockets.csproj`. Hence the build was failing. After I added DataMarketplace, the build succeeded. Would appreciate a way forward on this.

Reply:
It was renamed to Nethermind.Sockets: https://github.com/NethermindEth/nethermind/blob/master/src/Nethermind/Nethermind.Sockets/Nethermind.Sockets.csproj

but maybe it can be removed altogether from the sln file?

Reply:
That sounds good. I will test it out and make apt changes.

Reply:
A question: Do we have additional things to build on the CI? Maybe we can have all of them as part of a single workflow 🤷

Reply:
Benchmarks.sln
Cortex.sln
NDM.sln

Reply:
I think NDM.sln can be totally abandoned for now.

Reply:
Hi @LukaszRozmej 

I am running into this error while building `Benchmarks.sln`:

```sh
/Users/harshcasper/code/nethermind/nethermind/src/Nethermind/Nethermind.JsonRpc.Benchmark/EthModuleBenchmarks.cs(119,39): 
error CS1729: 'BlockchainBridge' does not contain a constructor that takes 11 arguments [/Users/harshcasper/code/nethermind/nethermind/src/Nethermind/Nethermind.JsonRpc.Benchmark/Nethermind.JsonRpc.Benchmark.csproj]
    156 Warning(s)
    1 Error(s)
```

Can you please suggest me what is going wrong here and how it can be fixed?

Reply:
Fixed in https://github.com/NethermindEth/nethermind/pull/3664 will be in master soon

Reply:
Merged to master, please update your branch

Reply:
Thanks!